src/Stryker.Core/Stryker.Core.UnitTest/Mutants/MutantOrchestratorTests.cs

```diff
@@ -500,6 +501,29 @@ public void MutationsShouldHaveLinespan()
             mutants.ShouldHaveSingleItem().Mutation.OriginalNode.GetLocation().GetLineSpan().StartLinePosition.Line.ShouldBe(2);
         }
 
+        [Theory]
+        [InlineData(@"static string Value { get; }
```

Comment:
This is a lot of lines for an argument. As the class is the same for both arguments you could move that to the method and keep the difference in the arguments.

Reply:
Good point. Updated.